docs/pages/getting-started/migrating-to-v5.mdx

```diff
@@ -125,11 +124,15 @@ export default async function Page() {
 
 Imports from `next-auth/react` are now marked with the [`"use client"`](https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive) directive. Therefore, they can be used in client components just like they were used in previous versions.
 
-<Callout type="warning">
-  If you have previously used `getSession()` or other imports from
-  `next-auth/react` server-side, you'll have to refactor them to use `auth()` as
-  shown above instead.
-</Callout>
+```ts filename="app/ClientComponent.tsx"
+import { useSession } from 'next-auth/react';
+
+const ClientComponent = () => {
+  const session = useSession();
+
+  return (<></>}
```

Comment:
There's a tiny syntax typo here, but also lets just give it a small use-case example.

```suggestion
  return (
	  <>
	  	<p>Welcome {session.user.name}</p>
	  </>
  )
```